UnitsNet.Tests/GeneratedCode/EnergyTestsBase.g.cs

```diff
@@ -54,31 +54,43 @@ public abstract partial class EnergyTestsBase
     {
         protected abstract double BritishThermalUnitsInOneJoule { get; }
         protected abstract double CaloriesInOneJoule { get; }
+        protected abstract double DecaeuThermsInOneJoule { get; }
```

Comment:
@anjdreas I would say that this should have a capital E since it is an abbreviation. What do you think? (Same with the others)

Reply:
I agree, this doesn't look quite right. These are probably generated by powershell and the `deca` prefix in JSON. `Deca` + `camelcase(EuTherms)`. This works for things like `Decigram` and `Millinewtonmeters`, but not for `DecieuTherm`. Not sure how we would fix this other than not using prefix and instead adding `DecaEuTherm` manually in JSON. Probably the easiest fix.

Reply:
On second thought, `DecaEuTherm` is probably not consistent with our existing naming either. Maybe `Decaeutherm`, or `Decaeurotherm` as `eurotherm` seems to be a wording I find when googling this.
I might prefer `Eurotherm` and `Decaeurotherm` after chewing on it a bit.

Reply:
@anjdreas Please also note that we also have `DecaimperialThermsInOneJoule`, `DecausThermsInOneJoule`. So to keep things simple at the moment, I would use ReSharper to rename this properties to `DecaEuThermsInOneJoule `, `DecaImperialThermsInOneJoule` and `DecaUsThermsInOneJoule`. What would say about it?

Reply:
The C# files get replaced when building so that would unfortunately not work. The "simplest" solution would be to add the deca units manually to the json file.